### PR TITLE
chore: Update Field VR tests to use new Field component

### LINKED ISSUE #26426: chore: Update Field VR tests to use new Field component
Once the new Field component has been implemented, need to update the VR tests to use the new component instead of the InputField, etc. shims.

For more details, see tracking issue:
* #26423

### PR DESCRIPTION
**Pull request by behowell**
## Previous Behavior

Individual Field components like `InputField`, etc. have been replaced by a single Field component, but the VR tests were still using the old individual components.

## New Behavior

Rewrite Field VR tests to use new standalone `Field` component.

## Related Issue(s)

* Fixes #26426

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d59cee0443fc357a3033d0e71e23659a2ce2d57d'>d59cee0443fc357a3033d0e71e23659a2ce2d57d</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/vr-tests-react-components/src/stories/Field.stories.tsx [hunk 1]
```diff
@@ -1,137 +1,165 @@
 import * as React from 'react';
```
Comment (fabricteam) [inline apps/vr-tests-react-components/src/stories/Field.stories.tsx:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 83 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26537&commitId=9172c2c0e0e4174d4e30fb315384e3ea0512f3a0&env=prod)

✅ There was 23 screenshots added, 60 screenshots removed, 1809 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 83 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.Checkbox-error.default.chromium.png|0| Added|
|Field.Combobox-error.default.chromium.png|0| Added|
|Field.Dropdown-error.default.chromium.png|0| Added|
|Field.ProgressBar-error.default.chromium.png|0| Added|
|Field.RadioGroup-error.default.chromium.png|0| Added|
|Field.Select-error.default.chromium.png|0| Added|
|Field.Slider-error.default.chromium.png|0| Added|
|Field.SpinButton-error.default.chromium.png|0| Added|
|Field.Switch-error.default.chromium.png|0| Added|
|Field.Textarea-error.default.chromium.png|0| Added|
|Field.base.default.chromium.png|0| Added|
|Field.hint.default.chromium.png|0| Added|
|Field.horizontal+longLabel.default.chromium.png|0| Added|
|Field.horizontal+noLabel.default.chromium.png|0| Added|
|Field.horizontal.default.chromium.png|0| Added|
|Field.required.default.chromium.png|0| Added|
|Field.size-large.default.chromium.png|0| Added|
|Field.size-small.default.chromium.png|0| Added|
|Field.validation-error.default.chromium.png|0| Added|
|Field.validation-none.default.chromium.png|0| Added|
|Field.validation-success.default.chromium.png|0| Added|
|Field.validation-warning.default.chromium.png|0| Added|
|Field.validationMessageIcon.default.chromium.png|0| Added|
|CheckboxField converged.base.default.chromium.png|0| Removed|
|CheckboxField converged.fieldLabel.default.chromium.png|0| Removed|
|CheckboxField converged.hint.default.chromium.png|0| Removed|
|CheckboxField converged.horizontal.default.chromium.png|0| Removed|
|CheckboxField converged.required.default.chromium.png|0| Removed|
|CheckboxField converged.size-large.default.chromium.png|0| Removed|
|CheckboxField converged.validation.default.chromium.png|0| Removed|
|ComboboxField converged.base.default.chromium.png|0| Removed|
|ComboboxField converged.hint.default.chromium.png|0| Removed|
|ComboboxField converged.horizontal.default.chromium.png|0| Removed|
|ComboboxField converged.required.default.chromium.png|0| Removed|
|ComboboxField converged.size-large.default.chromium.png|0| Removed|
|ComboboxField converged.size-small.default.chromium.png|0| Removed|
|ComboboxField converged.validation.default.chromium.png|0| Removed|
|InputField converged.base.default.chromium.png|0| Removed|
|InputField converged.hint.default.chromium.png|0| Removed|
|InputField converged.horizontal.default.chromium.png|0| Removed|
|InputField converged.required.default.chromium.png|0| Removed|
|InputField converged.size-large.default.chromium.png|0| Removed|
|InputField converged.size-small.default.chromium.png|0| Removed|
|InputField converged.validation.default.chromium.png|0| Removed|
|ProgressField converged.base.default.chromium.png|0| Removed|
|ProgressField converged.hint.default.chromium.png|0| Removed|
|ProgressField converged.horizontal.default.chromium.png|0| Removed|
|ProgressField converged.required.default.chromium.png|0| Removed|
|ProgressField converged.validation.default.chromium.png|0| Removed|
|RadioGroupField converged.base.default.chromium.png|0| Removed|
|RadioGroupField converged.hint.default.chromium.png|0| Removed|
|RadioGroupField converged.horizontal.default.chromium.png|0| Removed|
|RadioGroupField converged.required.default.chromium.png|0| Removed|
|RadioGroupField converged.validation.default.chromium.png|0| Removed|
|SelectField converged.base.default.chromium.png|0| Removed|
|SelectField converged.hint.default.chromium.png|0| Removed|
|SelectField converged.horizontal.default.chromium.png|0| Removed|
|SelectField converged.required.default.chromium.png|0| Removed|
|SelectField converged.size-large.default.chromium.png|0| Removed|
|SelectField converged.size-small.default.chromium.png|0| Removed|
|SelectField converged.validation.default.chromium.png|0| Removed|
|SliderField converged.base.default.chromium.png|0| Removed|
|SliderField converged.hint.default.chromium.png|0| Removed|
|SliderField converged.horizontal.default.chromium.png|0| Removed|
|SliderField converged.required.default.chromium.png|0| Removed|
|SliderField converged.validation.default.chromium.png|0| Removed|
|SpinButtonField converged.base.default.chromium.png|0| Removed|
|SpinButtonField converged.hint.default.chromium.png|0| Removed|
|SpinButtonField converged.horizontal.default.chromium.png|0| Removed|
|SpinButtonField converged.required.default.chromium.png|0| Removed|
|SpinButtonField converged.validation.default.chromium.png|0| Removed|
|SwitchField converged.base.default.chromium.png|0| Removed|
|SwitchField converged.hint.default.chromium.png|0| Removed|
|SwitchField converged.horizontal.default.chromium.png|0| Removed|
|SwitchField converged.required.default.chromium.png|0| Removed|
|SwitchField converged.validation.default.chromium.png|0| Removed|
|TextareaField converged.base.default.chromium.png|0| Removed|
|TextareaField converged.hint.default.chromium.png|0| Removed|
|TextareaField converged.horizontal.default.chromium.png|0| Removed|
|TextareaField converged.required.default.chromium.png|0| Removed|
|TextareaField converged.size-large.default.chromium.png|0| Removed|
|TextareaField converged.size-small.default.chromium.png|0| Removed|
|TextareaField converged.validation.default.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>

Comment (behowell) [reply apps/vr-tests-react-components/src/stories/Field.stories.tsx:1]: Changes look good to me: adding all new tests for Field, and removing the individual component tests like InputField.

Comment (behowell) [reply apps/vr-tests-react-components/src/stories/Field.stories.tsx:1]: Note that the ProgressBar with error state doesn't show a red bar. That is expected until FieldContext is added and integrated with ProgressBar: #26538

**Review by khmakoto — APPROVED** [review]

File: apps/vr-tests-react-components/src/stories/Field.stories.tsx (hunk 1 above)
Comment (behowell) [reply apps/vr-tests-react-components/src/stories/Field.stories.tsx:1]: We're no longer planning to add the FieldContext from #26538, so I've updated the VR tests here to just add `size` to the `<Input>` for the size test, and  `validationState` to the `<ProgressBar>` for its test.

**Pull request merged**